internal/params/flags.go

```diff
@@ -33,6 +33,8 @@ const (
 	BranchFlag                   = "branch"
 	BranchFlagSh                 = "b"
 	ScanIDFlag                   = "scan-id"
+	CodeRepositoryFlag           = "code-repository-url"
```

Comment:
didn't it changed to repository-sever-url??

Reply:
waiting for product to answer....